### PR TITLE
docs: note regarding emit with backticks

### PR DESCRIPTION
**Pull request by ringabout**

### CODE REVIEW

**Review by dom96 — COMMENTED** [review]
Nitpick: this is more a note, rather than a warning. Do we support those?

**Comment by ghost** [comment]
> Nitpick: this is more a note, rather than a warning. Do we support those?

we do, they're green instead of yellow :)

File: doc/manual.rst [hunk 1]
```diff
@@ -7039,9 +7039,9 @@ Example:
   void fun(){}
   """.}
 
-For backward compatibility, if the argument to the `emit` statement
-is a single string literal, Nim symbols can be referred to via backticks.
-This usage is however deprecated.
+.. warning:: For backward compatibility, if the argument to the `emit` statement
```
Comment (ringabout) [inline doc/manual.rst]:
```suggestion
.. note:: For backward compatibility, if the argument to the `emit` statement
```

**Comment by ringabout** [comment]
Github Action CI is fragile

image

**Review by timotheecour — APPROVED** [review]
LGTM, however we should really fix the underlying bugs:
* A1 from https://github.com/nim-lang/Nim/issues/13943
* https://github.com/nim-lang/Nim/issues/3791
because backticks are IMO more readable than the array version

**Pull request merged**